CodeEdit/Features/Settings/Pages/SourceControlSettings/SourceControlGeneralView.swift

```diff
@@ -32,6 +35,14 @@ struct SourceControlGeneralView: View {
                 defaultBranchName
             }
         }
+        .onAppear {
+            Task {
+                defaultBranch = try await gitConfig.get(key: "init.defaultBranch", global: true) ?? ""
+                Task {
```

Comment:
Is this extra Task a duplicate or is there a purpose for it?

Reply:
This delayed setting this variable just enough to where the onChange wasn't called before this initial value was set.

Reply:
I just was being consistent with to other implementation possibly in anticipation of adding a Toggle down the road

Reply:
Added `try? await Task.sleep(for: .milliseconds(0))` instead.